### DISCUSSION TITLE
Calendar days to working days for cycle time

### DISCUSSION THREAD

**Discussion by Ellejaypea** [nbrown02/FlowViz#146] [Q&A] [answered]
Hi! I love using FlowViz to better help understand how my teams are working, however we are struggling with cycle time being measured in calendar days and not working days. I know ADO has a concept of what working days are, so can you advise on how we could switch it out to show working days instead?

**Comment by nbrown02** [comment, marked as answer]
Hey @Ellejaypea - thanks for reaching out. 

Microsoft does the CycleTime and LeadTime calculations for us, rather than it be done by myself, so it's really quite tricky. 
Going this route adds unnecessary complexity - what about holidays? what about holidays for one country and not another? what about when someone was assigned an item and was on annual leave? etc. 

In all honesty I'm not sure how you'd do it (I'd probably ask ChatGPT!) and for the time you'd need to invest in calculating (and maintaining) this it's really not going to be worth it for the value it would add...
